it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersSoleProprietorUtrControllerISpec.scala

```diff
@@ -5,64 +5,64 @@ import play.api.libs.ws.DefaultWSCookie
 import play.api.test.WsTestClient
 import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
 
-class BusinessPartnersCorporateBodyUtrControllerIntegrationSpec
+class BusinessPartnersSoleProprietorUtrControllerISpec
   extends Specifications with TestConfiguration {
 
-  "GET /form/business-partners/corporate-body-corporation-tax-unique-taxpayer-reference" should {
+  val route = "/business-partners/partner-self-assessment-unique-taxpayer-reference"
+
+  s"GET /business-partners/partner-self-assessment-unique-taxpayer-reference" should {
```

Comment:
could use the route here like you've done in the CorporateBodyISpec

Reply:
Also its missing the /form/... prefix

Reply:
form prefix not required, confirmed by James/Ola.